extensions/vscode/CONTRIBUTING.md

```diff
@@ -85,3 +85,71 @@ npm test
 
 [conventional_commits_link]: https://www.conventionalcommits.org/en/v1.0.0
 [bug_report_link]: https://github.com/dart-frog-dev/dart_frog/issues/new?assignees=&labels=bug&template=bug_report.md&title=fix%3A+
+
+
+## 🐸 Releasing Dart Frog’s extension on [VS Code Marketplace](https://marketplace.visualstudio.com/vscode)
+
+1. Go to the **main** branch and ensure it is up to date with the remote (from extensions/vscode):
+
+```bash
+git checkout main
+git pull
+```
+
+2. Run the script that will generate the CHANGELOG for you (from extensions/vscode): 
+
+```bash
+sh ../../tool/release_ready.sh <new-version>
+```
+
+`<new-version>`: The version of this new extension release, for example: 0.2.1
+
+The [*release_ready*](https://github.com/dart-frog-dev/dart_frog/blob/vscode-v0.2.1/tool/release_ready.sh) script will:
+
+- Create a new branch just for this release and checkout to it.
```

Comment:
Is a new branch needed? I think it should be sufficient to tag in most cases.

Reply:
Oh I misunderstood! You meant checkout a branch just to make the change and open a pull request 👍